backend/src/main/java/gov/cdc/usds/simplereport/db/model/Organization.java

```diff
@@ -11,6 +11,9 @@
   @Column(nullable = false, unique = true)
   private String organizationName;
 
+  @Column(nullable = false)
```

Comment:
**question:** I noticed that `not null` is _not_ set on this field in the migration. Just curious: are we planning to backfill the data and then enforce this requirement there as well?